.github/workflows/manual.yml

```diff
@@ -12,16 +12,12 @@ on:
 
 # A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  perform:
+  deployproject:
     strategy:
       matrix:
         repo_name:
-          # define the list of repos here
-          - codesnippet
-          - colorbutton
-          - fakeobjects
-          - font
-          - notification
+        # define the list of repos here
+        - templates
```

Comment:
Looks like we can revert changes here and just add `templates` to the list. And keep alphabetical order)

Reply:
Resolved: https://github.com/drupal-ckeditor-libraries-group/actions_runner/pull/4/commits/f81944eb6cc769c253af60876128c9531403dc99